source/Preferences.cpp

```diff
@@ -182,6 +185,22 @@ bool Preferences::ZoomViewOut()
 
 
 
+bool Preferences::ToggleScreenMode()
+{
+	GameWindow::ToggleFullscreen();
+	return true;
+}
```

Comment:
No need for this one to return a bool, if it always returns true. (see ToggleAmmoUsage())
Don't forget to change it in the .h, too.

As you can see, the fact this returns a bool is not actually used by your code in the PreferencesPanel.

In general, if it is an action that cannot fail, it should be void. If it sometimes does not actually do anything (for instance ToggleVSync()) then its a bool that is returned.